src/Artifacts.jl

```diff
@@ -59,6 +59,25 @@ function soil_params_artifact_folder_path(; context = nothing)
     return @clima_artifact("soil_params_Gupta2020_2022", context)
 end
 
+"""
+    soil_grids_params_artifact_path(; lowres = true, context)
+
+Return the path to the folder that contains the soil texture parameters
+needed for the Balland and Arp (2005) thermal conductivity model.
+
+Returns a ~1 degree version by default (lowres = true).
+"""
+function soil_grids_params_artifact_folder_path(;
+    context = nothing,
+    lowres = true,
+)
+    if lowres
+        return @clima_artifact("soilgrids_lowres", context)
+    else
+        return @clima_artifact("soilgrids", context)
+    end
+end
+
```

Comment:
If there is only one file in each artifact, we should probably follow the existing pattern and make this 

`soil_grids_params_artifact_data()` or `soil_grids_params_data_path`

and directly return the path to the lowres or full-resolution artifact.  See the `topmodel_data_path` and `lehmann2008_evaporation_data` functions.

Reply:
ok! I fixed this